### PR TITLE
Add option to remove "type" attribute #366

### PR DESCRIPTION
**Pull request by ovidijusr**
Issue: https://github.com/webpack-contrib/style-loader/issues/366

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@ovidijusr thanks, can you describe use case?

**Comment by ovidijusr** [comment]
Well personally, I would set not having a "type" tag as default behaviour. Because [w3c markup validator](https://validator.w3.org/) gives warnings about that type. But this is breaking tests and i'm not sure if that won't affect something else.
Being said that, giving an option for this is a win/win because you can still use old behaviour and opt in for w3c correct one. [cssinjs had a similar case](https://github.com/cssinjs/jss/issues/724)

TLDR; if you use w3c - disabling type attribute helps to solve w3c warnings

**Comment by evilebottnawi** [comment]
@ovidijusr what about remove this by default?

**Comment by ovidijusr** [comment]
@evilebottnawi  Sure, should i update PR?

**Comment by evilebottnawi** [comment]
@ovidijusr yep, let's do it

**Comment by evilebottnawi** [comment]
Close in favor https://github.com/webpack-contrib/style-loader/pull/384
